ToDoApp/ToDoApp.xcodeproj/xcuserdata/tvuong.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
since the gitignore file was added to the project after the project commit. this file has. been tracked by git, but it is user settings file which should not be tracked by git. in order to untrack it you can use the following command:
`git rm --cached ToDoApp/ToDoApp.xcodeproj/xcuserdata/tvuong.xcuserdatad/xcschemes/xcschememanagement.plist`

Reply:
and the same is needed for ToDoApp/ToDoApp.xcodeproj/project.xcworkspace/xcuserdata/tvuong.xcuserdatad/UserInterfaceState.xcuserstate
`git rm --cached ToDoApp/ToDoApp.xcodeproj/project.xcworkspace/xcuserdata/tvuong.xcuserdatad/UserInterfaceState.xcuserstate`